test/sai_test/sai_lag_test.py

```diff
@@ -34,29 +34,33 @@ def setUp(self):
         T0TestBase.setUp(self)
 
     def load_balance_on_src_ip(self):
-        ip_dst = self.lag1_route_dst
+        sai_thrift_create_router_interface(self.client,
```

Comment:
No need to recreate this interface for port1 cause it has been create in route_configer.py